src/r2b2/audit.py

```diff
@@ -95,6 +95,7 @@ def __init__(self, alpha: float, beta: float, max_fraction_to_draw: float, repla
         self.sample_winner_ballots = []
         self.risk_schedule = []
         self.stopping_prob_schedule = []
+        self.pvalue_schedule = []
```

Comment:
Make sure this gets reinitialized to empty in __reset()

Reply:
Noted, thanks!